spec/atom-cfn-lint-spec.js

```diff
@@ -17,17 +17,31 @@ permissions and limitations under the License.
 
 import * as path from 'path';
 
-const badYamlPath = path.join(__dirname, 'fixtures', 'bad.yaml');
-const goodYamlPath = path.join(__dirname, 'fixtures', 'good.yaml');
-const nonCloudFormationYamlPath = path.join(__dirname, 'fixtures', 'not_cloudformation.yaml');
-const badJsonPath = path.join(__dirname, 'fixtures', 'bad.json');
-const goodJsonPath = path.join(__dirname, 'fixtures', 'good.json');
-const nonCloudFormationJsonPath = path.join(__dirname, 'fixtures', 'not_cloudformation.json');
+const fixturesDir = path.join(__dirname, 'fixtures')
+
+const fixtures = {
+  goodJson: [ 'templates', 'good.json'],
+  goodYaml: [ 'templates', 'good.yaml'],
+  nonCfnYaml: [ 'templates', 'not_cloudformation.yaml'],
+  nonCfnJson: [ 'templates', 'not_cloudformation.json'],
+  badJson: [ 'templates', 'bad.json'],
+  badYaml: [ 'templates', 'bad.yaml'],
+  overrideYaml: [ 'templates', 'override.yaml'],
+  overrideSpec: [ 'override_spec', 'required.json'],
+  appendRules: [ 'append_rules'],
+}
+
+const paths = Object.keys(fixtures)
+  .reduce((accumulator, fixture) => {
+    const acc = accumulator
+    acc[fixture] = path.join(fixturesDir, ...(fixtures[fixture]))
+    return acc
+  }, {})
 
 describe('The CFN Lint provider for Linter', () => {
   const { lint } = require(path.join('..', 'lib', 'atom-cfn-lint.js')).provideLinter();
 
-  beforeEach(() => {
+  beforeEach(async () => {
```

Comment:
Since you seem to be using the built in Jasmine 1.3 test runner, `async` isn't supported (even with `babel`). You'll either need to write your own fixer for it, or you can use `jasmine-fix` [like this](https://github.com/AtomLinter/linter-jshint/blob/a3712b7295dc038d685e6b65f6e67db3f4611f37/spec/linter-jshint-spec.js#L3-L4) in order for this to work.

If you don't then the specs all will "pass" but not actually be doing anything.